tidy3d/components/simulation.py

```diff
@@ -1009,6 +1010,22 @@ def _validate_monitor_size(self) -> None:
                 f"a maximum of {MAX_SIMULATION_DATA_SIZE_GB:.2f}GB are allowed."
             )
 
+        # Some monitors store much less data than what is needed internally. Make sure that the
+        # internal storage also does not exceed the limit.
+        for monitor in self.monitors:
+            num_cells = self._monitor_num_cells(monitor)
+            # intermediate storage needed
+            internal_data_gb = monitor._storage_size_solver(num_cells=num_cells, tmesh=self.tmesh)
```

Comment:
would be a bit easier to understand if we named the variables like
```py
internal_data_bytes = monitor._storage_size_solver(num_cells=num_cells, tmesh=self.tmesh)
internal_data_gb = internal_data_bytes / 2**30
```
also in the other instance where the 2 ** 30 is divided out.